test/lib/util/fast_json_encoder_test.exs

```diff
@@ -0,0 +1,61 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.FastJasonEncoderTest do
+  use Croma.TestCase
+  use ExUnitProperties
+
+  def validate(value) do
+    validate(value, value)
+  end
+  def validate(source, expected) do
+    {:ok, json} = FastJasonEncoder.encode(source)
+    assert json == Poison.encode!(source)
+    assert expected == Poison.decode!(json)
+  end
+
+  test "primitives" do
+    values = [0, 1, 1.5, true, false, nil]
+    Enum.each(values, &validate(&1))
+    validate(:ok, "ok") # Atom is encoded as a string
+  end
+
+  test "complext types" do
+    values = [[], [1.5], [1, "foo"], %{}, %{"key" => "value"}]
+    Enum.each(values, &validate(&1))
+
+    # Atom is encoded as a string
+    validate(%{key: "value"}, %{"key" => "value"})
+    validate(%{key: :ok    }, %{"key" => "ok"   })
+
+    # Skip FastJasonEncoder.encode vs Poison.encode because the order of the field may change
```

Comment:
The form `function/arity_num` is a general representation of functions in Elixir.
```suggestion
    # Skip `FastJasonEncoder.encode/1` vs `Poison.encode/1` because the order of the field may change.
```